pkg/kubelet/kubelet.go

```diff
@@ -1731,6 +1796,78 @@ func (kl *Kubelet) getPullSecretsForPod(pod *api.Pod) ([]api.Secret, error) {
 	return pullSecrets, nil
 }
 
+// listPodsFromCgroupMounts scans through all the subsytems of pod cgroups
+// Get list of pods whose cgroup still exist on the cgroup mounts
+func (kl *Kubelet) listPodsFromCgroupMounts() (map[types.UID]string, error) {
```

Comment:
A refactor in a separate PR is fine by me as long as it doesn't fall through the cracks.